cmd/mattermost/commands/config.go

```diff
@@ -367,6 +376,93 @@ func UpdateMap(configMap map[string]interface{}, configSettings []string, newVal
 	}
 }
 
+func configResetCmdF(command *cobra.Command, args []string) error {
+	configStore, err := getConfigStore(command)
+	if err != nil {
+		return err
+	}
+
+	defaultConfig := &model.Config{}
+	defaultConfig.SetDefaults()
+
+	confirmFlag, _ := command.Flags().GetBool("confirm")
+	if confirmFlag {
+		if _, err = configStore.Set(defaultConfig); err != nil {
+			return errors.Wrap(err, "failed to set config")
+		}
+		configStore.Set(defaultConfig)
```

Comment:
This line seems to be a duplicate as the `configStore.Set(defaultConfig)` operation has just been done and error checked just above.

Reply:
My bad that I missed this, thank you very much for spotting this @mgdelacroix ! Should be fine by now! :slightly_smiling_face:

Reply:
No worries! Thanks for the fix!! :)